macroslib/src/cpp/mod.rs

```diff
@@ -533,6 +537,17 @@ fn merge_rule(ctx: &mut CppContext, mut rule: TypeMapConvRuleInfo) -> Result<()>
     Ok(())
 }
 
+fn cache_f_code(cached_code: &mut FxHashSet<(SmolStr, String)>, f_module_name: &SmolStr, f_code_str: &str) -> bool {
+    for (module_name, code) in cached_code.iter() {
```

Comment:
The idea of HashSet, that it has method contains,
that works with O(1) instead of loop that works O(n):

https://doc.rust-lang.org/std/collections/struct.HashSet.html#method.contains